### DISCUSSION TITLE
Can I call a build or make script/command with a key combo?

### DISCUSSION THREAD

**Discussion by gingerbeardman** [zed-industries/zed#7584] [Feature Requests]
I'd like to be able to press Cmd+R and run my project build script. 

Is this currently possible?

**Comment by SomeoneToIgnore** [comment]
Not possible currently, hopefully next month, thanks to @osiewicz : https://github.com/zed-industries/zed/issues/7108

**Reply by hw2007** [reply]
I'm excited for this feature!!!!
